src/sentry/incidents/subscription_processor.py

```diff
@@ -444,9 +444,8 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
             # TODO: Delete subscription here.
             return
 
-        # TODO:
-        # if AlertRule.monitor_type === ACTIVATED and AlertRule.is_expired():
-        #       delete query in snuba
+        # Update each alert_rule based on the subscription update
+        self.alert_rule.process_hooks(self.subscription)
```

Comment:
@wedamija how do you feel about this hook?

I know you mentioned that the `AlertRule` shouldn't have any ties to the SnubaQueries, so want to double check to make sure this aligns with that separation of concerns 🙏